hadoop-hdds/common/src/test/java/org/apache/hadoop/ozone/container/ContainerTestHelper.java

```diff
@@ -436,6 +441,7 @@ public static Builder newPutBlockRequestBuilder(Pipeline pipeline,
 
     ContainerProtos.PutBlockRequestProto.Builder putRequest =
         ContainerProtos.PutBlockRequestProto.newBuilder();
+    putRequest.setEof(true);
```

Comment:
Why is `eof=true` needed here?  I think caller can set it later.

Reply:
Ah, I think I added when debugging.

Reply:
Ah, I think I added this while testing.